x-pack/plugins/observability_solution/entity_manager/server/lib/entities/templates/entities_history_template.ts

```diff
@@ -33,7 +33,7 @@ export const getEntitiesHistoryIndexTemplateConfig = (
     ENTITY_EVENT_COMPONENT_TEMPLATE_V1,
     ...getCustomHistoryTemplateComponents(definitionId),
   ],
-  index_patterns: [`${ENTITY_HISTORY_INDEX_PREFIX_V1}.*`],
+  index_patterns: [`${ENTITY_HISTORY_INDEX_PREFIX_V1}.${definitionId}.*`],
```

Comment:
should use the utils from https://github.com/elastic/elastic-entity-model/issues/108 once merged

Reply:
i think the final `.` should be removed in order for this to match the `latest` datastreams

Reply:
ah this is history only ... nvm

Reply:
yes note that the dot is important here otherwise we would match definitions with similar prefixes like in the test